app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :users
   belongs_to :posts
+
+  after_save :update_comments_counter
+
+  private
+
+  def update_comments_counter
+    posts.increment!(:update_comments_counter)
```

Comment:
- [ ] I think you meant to increment the **`:comments_counter`**, not `:update_comments_counter`. Kindly fix that.